torchgeo/datasets/utils.py

```diff
@@ -482,3 +483,30 @@ def rgb_to_mask(
         if isinstance(cmask, np.ndarray):
             mask[cmask.all(axis=-1)] = i
     return mask
+
+
+def contrast_stretch(
```

Comment:
@calebrob6 this looks similar to the function you are adding in #251 but with `lower=0, upper=100`. Maybe we should wait on #251 and use that function here?

Reply:
Removed `contrast_stretch` and switched to `percentile normalization`